backend/src/routes/projectRouter.ts

```diff
@@ -225,15 +225,29 @@ projects.get("/v1/projects", async (req, res, next) => {
 });
 
 // Returns the session user's projects
-projects.get("/v1/members", async (req: Request, res, next) => {
-  if (req.query["user"]) {
-    const userId = req.query["user"];
-    const userMembers = await projectController.findUserProjects(
-      userId.toString()
-    );
-    res.json(userMembers);
+projects.get(
+  "/v1/members",
+  (req, res, next) => {
+    if (req.query["user"]) {
+      next();
+    } else {
+      res.status(401).json({ errors: ["unauthenticated"] });
+    }
+  },
+  async (req: Request, res, next) => {
+    try {
+      if (req.query["user"]) {
```

Comment:
For the last handler of the API request, don't forget to match the if block with an else block as well, or else an API call without the `user` parameter will not return and timeout the frontend.